CMakeLists.txt

```diff
@@ -0,0 +1,91 @@
+
+# Copyright (C) 2021 Istituto Italiano di Tecnologia (IIT). All rights reserved.
+# This software may be modified and distributed under the terms of the
+# GNU Lesser General Public License v2.1 or any later version.
+
+cmake_minimum_required(VERSION 3.14)
```

Comment:
Should we bump this already to a newer version? I guess we can start at least with 3.16, if not newer.